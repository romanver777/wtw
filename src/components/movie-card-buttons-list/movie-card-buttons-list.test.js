/* eslint-disable @typescript-eslint/no-unsafe-call */
/* eslint-disable @typescript-eslint/no-unsafe-member-access */
/* eslint-disable @typescript-eslint/no-unsafe-assignment */
import React from "react";
import TestRenderer from "react-test-renderer";
import MovieCardButtonsList from "../movie-card-buttons-list/movie-card-buttons-list";

describe("MovieCardButtonsList component", () => {
  const props = {
    children: [
      <div className="className" key="someKey"></div>,
      <div className="another className" key="anotherKey"></div>,
    ],
  };
  const Trenderer = TestRenderer.create(<MovieCardButtonsList {...props} />);
  it("renders correctly", () => {
    expect(Trenderer.toJSON()).toMatchSnapshot();
  });
});
