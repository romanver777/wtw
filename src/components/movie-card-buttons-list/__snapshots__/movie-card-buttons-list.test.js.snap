// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCardButtonsList component renders correctly 1`] = `
<div
  className="movie-card__buttons"
>
  <div
    className="className"
  />
  <div
    className="another className"
  />
</div>
`;
